tools/testkit/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#![allow(dead_code)]
+
+use fvm_integration_tests::dummy::DummyExterns;
+use fvm_integration_tests::tester::{Account as TAccount, Tester};
+use fvm_ipld_blockstore::MemoryBlockstore;
+use fvm_shared::state::StateTreeVersion;
+use fvm_shared::version::NetworkVersion;
+
+pub mod bundle;
+pub mod fevm;
+
+/// Execution options
+pub struct ExecutionOptions {
+    /// Enables debug logging
+    pub debug: bool,
+    /// Enables gas tracing
+    pub trace: bool,
+    /// Enabls events
+    pub events: bool,
+}
+
+pub type BasicTester = Tester<MemoryBlockstore, DummyExterns>;
+pub struct Account {
+    pub account: TAccount,
+    pub seqno: u64,
+}
+
+pub fn new_tester(bundle_path: String) -> BasicTester {
+    let blockstore = MemoryBlockstore::default();
+    let bundle_cid = match bundle::import_bundle(&blockstore, bundle_path.as_str()) {
+        Ok(cid) => cid,
+        Err(what) => {
+            exit_with_error(format!("error loading bundle: {}", what));
+        }
+    };
+    Tester::new(
+        NetworkVersion::V18,
+        StateTreeVersion::V5,
+        bundle_cid,
+        blockstore,
+    )
+    .unwrap_or_else(|what| {
+        exit_with_error(format!("error creating execution framework: {}", what));
```

Comment:
It's surprising to see `sys::exit` in a library function.

Reply:
yeah, I can remove it and just panic I guess, depending on how strongly you feel on this guys.
Not really frettting however.

Reply:
turned to panics.